src/quo2/components/messages/system_message.cljs

```diff
@@ -155,12 +161,12 @@
                 :color   (or customization-color :primary)
                 :opacity 5}
     :timestamp timestamp}
-   [rn/view
+   [rn/view {:style {:flex-shrink 1}}
     [rn/view
      {:flex-direction :row
-      :flex-wrap      :wrap
-      :height         18.2}
```

Comment:
@ibrkhalil I've noticed the addition of this `height` in [your PR here](https://github.com/status-im/status-mobile/pull/16997/files#diff-07804a4aa53d86bfec5907032607dc58b11e0e4a1a1d5801625c0be2ad27f738R162). I'm uncertain about its purpose; please advise if its removal could cause any problems.